spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/FindMultipleInstantiationsOfSingletons.java

```diff
@@ -0,0 +1,214 @@
+package edu.umd.cs.findbugs.detect;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+
+public class FindMultipleInstantiationsOfSingletons extends AbstractIntegrationTest {
```

Comment:
The naming convention for the test classes is the tested class with the `Test` suffix. Can you please rename this class?